src/PointwiseFunctions/AnalyticData/Xcts/Binary.hpp

```diff
@@ -215,7 +218,15 @@ class Binary : public ::AnalyticData<3, Registrars> {
     using type = std::unique_ptr<IsolatedObjectBase>;
   };
   struct AngularVelocity {
-    static constexpr Options::String help = "Orbital angular velocity";
+    static constexpr Options::String help =
+        "Orbital angular velocity 'Omega0' about the z-axis. Added to the "
+        "background shift as a term 'Omega0 x r'.";
+    using type = double;
+  };
+  struct RadialVelocity {
```

Comment:
This parameter is not the radial velocity (it is the radial velocity divided by the radius, or the initial time derivative of the scale factor)